index.html

```diff
@@ -8,7 +8,14 @@
 	<!--Header-->
 	<section>
 		<header class="Selection-header">
-			Header
+			<div >
+				<select class="Selection-header-dropDown">
```

Comment:
When you know that you will use nested classes, you may want to try something like `Selection__header-dropDown`. And so in your header above, it will be `Selection__header`.